### PR TITLE
Added unified api support to iOS

### PR DESCRIPTION
**Pull request by monday8am**
- Added support to Unified API.
- Updated Newtonsoft.Json to version 6.0.8

### CODE REVIEW

File: Bugsnag.iOS/Data/TouchSystemInfo.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ internal class TouchSystemInfo : SystemInfo
         public string Model { get; set; }
 
         [JsonProperty ("diskSize", DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public ulong DiskSize { get; set; }
+		public ulong DiskSize { get; set; }
```
Comment (roosmaa) [inline Bugsnag.iOS/Data/TouchSystemInfo.cs]: What was wrong with spaces?

File: Bugsnag.iOS/StateReporter.cs [hunk 2]
```diff
@@ -55,7 +56,7 @@ public TouchSystemInfo GetSystemInfo ()
                 Id = client.DeviceId,
                 Manufacturer = "Apple",
                 Model = Model,
-                ScreenDensity = UIScreen.MainScreen.Scale,
+				ScreenDensity = (float)UIScreen.MainScreen.Scale,
```
Comment (roosmaa) [inline Bugsnag.iOS/StateReporter.cs]: Wrong indention again.

**Comment by roosmaa** [comment]
Other than the wrong intent settings, looks ok..

**Comment by roosmaa** [comment]
Did you run the test solutions? For me iOS crash tests don't work anymore, Android works as expected.

**Comment by roosmaa** [comment]
Ok, seems there has been a bug in the tests before this branch... UGH!

[CrashTestViewController.cs:57](https://github.com/toggl/bugsnag-xamarin/blob/master/Tests.iOS/CrashTestViewController.cs#L57) should read instead: `if (bugsnagClient == null) {`.

Can you fix my bug please? :)

File: Bugsnag.iOS/StateReporter.cs (hunk 2 above)
Comment (monday8am) [inline Bugsnag.iOS/StateReporter.cs]: Github don't like tab indent ..I'll fix it

Comment (roosmaa) [inline Bugsnag.iOS/StateReporter.cs]: It's not Github, it's me :) since the project uses spaces to indent it's bad to mix tabs into them.

**Comment by monday8am** [comment]
now looks better

https://github.com/toggl/bugsnag-xamarin/blob/ecb5c350e7c7b76450cb21d0c97973c05023f12f/Bugsnag.iOS/StateReporter.cs

:)

File: Bugsnag.iOS/Data/TouchSystemInfo.cs [hunk 3]
```diff
@@ -11,7 +11,7 @@ internal class TouchSystemInfo : SystemInfo
         public string Model { get; set; }
 
         [JsonProperty ("diskSize", DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public ulong DiskSize { get; set; }
+	    public ulong DiskSize { get; set; }
```
Comment (roosmaa) [inline Bugsnag.iOS/Data/TouchSystemInfo.cs]: I still see a tab!

Comment (monday8am) [inline Bugsnag.iOS/Data/TouchSystemInfo.cs]: :weary: 

https://github.com/toggl/bugsnag-xamarin/blob/unified-api-support/Bugsnag.iOS/Data/TouchSystemInfo.cs

**Pull request merged**